docs/modules/airflow/examples/example-airflow-gitsync.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: airflow.stackable.tech/v1alpha1
+kind: AirflowCluster
+metadata:
+  name: airflow
+spec:
+  image:
+    productVersion: "2.4.1"
+    stackableVersion: "23.4.0-rc3"
+  executor: CeleryExecutor
+  loadExamples: false
+  exposeConfig: false
+  credentialsSecret: test-airflow-credentials # <1>
+  clusterConfig:
+    dagsGitSync: # <2>
+      - name: git-sync # <3>
```

Comment:
Whats the benefit of having the user come up with a name rather than enumerating it internally in the operator? Sorry if this is obvious from some code below, I'm just looking at the CRD example

Reply:
Good point. I've removed the name field and assign it internally instead.